source/tests/pt/modifier/test_data_modifier.py

```
@@ -181,21 +189,69 @@ def modify_data(self, data: dict[str, Array | float], data_sys: DeepmdData) -> N
 
 @BaseModifier.register("scaling_tester")
 class ModifierScalingTester(BaseModifier):
-    def __new__(cls, *args, **kwargs):
-        return super().__new__(cls)
+    def __new__(
+        cls,
+        *args: tuple,
```

Ruff (0.14.11)

Unused static method argument: `args`

(ARG004)